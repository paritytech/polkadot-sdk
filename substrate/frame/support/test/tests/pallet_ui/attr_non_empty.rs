#[frame_support::pallet [foo]]
mod foo {
}

fn main() {
}
