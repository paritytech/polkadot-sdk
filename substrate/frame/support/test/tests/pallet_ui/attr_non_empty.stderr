error: Invalid pallet macro call: expected no attributes, e.g. macro call must be just `#[frame_support::pallet]` or `#[pallet]`
 --> $DIR/attr_non_empty.rs:1:26
  |
1 | #[frame_support::pallet [foo]]
  |                          ^^^
